src/cleanfigure.m

```diff
@@ -251,7 +246,7 @@ function pruneOutsideBox(meta, handle)
       id_remove  = id_remove(~idx);
   end
   % Replace the data points
-  replaceData(handle, id_replace, NaN(length(id_replace), size(data,2)));
```

Comment:
That's fine :smile:. 
*(I though we might have used that in our `pixelate` function as well, but it seems not, and probably it's not any easier to do so).*